sanitiser/_details.js

```diff
@@ -0,0 +1,32 @@
+// validate inputs, convert types and apply defaults
+function sanitize( req, default_value ){
+  
+  var clean = req.clean || {};
+  var params= req.query;
+
+  if (default_value === undefined) {
+    default_value = true;
+  }
+  
+  default_value = !!default_value;
+
+  // ensure the input params are a valid object
+  if( Object.prototype.toString.call( params ) !== '[object Object]' ){
```

Comment:
I started using `npm is-object` for this `var isObject = require('is-object');` https://github.com/pelias/api/blob/master/package.json#L41; because javascript